src/toast-error/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+   <translation id="toast_error_an_error_occured" qtlid="148981">Une erreur est survenue.</translation>
```

Comment:
⚠️ reminder: our internal script must be launched.

Reply:
I've added `translation required` label